GitUtils.ps1

```diff
@@ -159,13 +159,19 @@ function Get-GitStatus($gitDir = (Get-GitDirectory)) {
 
                 $indexAdded.AddRange($castStringSeq.Invoke($null, (,@($cacheResponse.IndexAdded))))
                 $indexModified.AddRange($castStringSeq.Invoke($null, (,@($cacheResponse.IndexModified))))
-                $indexModified.AddRange($castStringSeq.Invoke($null, (,@($cacheResponse.IndexRenamed | Select-Object -ExpandProperty Old))))

```

Comment:
Cast was choking on the System.Management.Automation.PSObject returned by Select-Object.